examples/howto_examples/cerebellar_peduncles.py

```diff
@@ -0,0 +1,160 @@
+"""
+================================
+Delineating cerebellar peduncles
+================================
+
+The cerebellar peduncles are white matter tracts that connect the cerebellum to
+the brainstem and cortex. In this example, we show how to delineate the
+cerebellar peduncles using a subject from the Healthy Brain Network dataset.
+
+This how-to will focus on the definition of the Cerebellar Peduncles (CP) based
+on [1]_, [2].
+
+"""
+
+import AFQ.data.fetch as afd
+import AFQ.api.bundle_dict as abd
+from AFQ.api.group import GroupAFQ
+from AFQ.definitions.image import RoiImage, ImageFile
+
+
+"""
+We will use a subject from the HBN dataset. When considering the data
+for this operation, look to see whether the acquisition volume includes the
+cerbellum. If it does not, it will be hard to delineate the CPs.
+"""
+
+bids_path = afd.fetch_hbn_afq(["NDARAA948VFH"])[1]
+
+"""
+The next line downloads the cerebellar peduncle templates from `Figshare <https://figshare.com/articles/dataset/Regions_of_interest_for_automated_fiber_quantification_of_cerebellar_bundles/23201630>`_.
+
+"""
+
+cp_rois = afd.read_cp_templates()
+
+
+"""
+The following lines define a bundle dictionary for the cerebellar
+peduncles. There are three CPs: The ICP, the MCP, and the SCP. Each CP is
+defined by two inclusion ROIs and one exclusion ROI. The Inferior CPs are
+defined by inclusion ROIs. They do not decussate, so "cross_midline" is set to
+False. The Superior CPs are defined by two inclusion ROIs and an exclusion ROI,
+where each SCP's most superior inclusion ROI is the other SCP's exclusion ROI.
+They decussate, so "cross_midline" is set to True. The Middle CPs are defined
+by two inclusion ROIs and they use the SCP intermediate ROIs as exclusion ROIs.
+"""
+
+cp_bundles = abd.BundleDict(
```

Comment:
add this bundle dict as a constant in api.bundle_dict file (CerebellarBundleDict)

Reply:
Do you mean as a derived class from BundleDict? Along the lines of the PediatricBundleDict

Reply:
Hmmm.... Maybe a better idea is to have a function which returns this.
```
def get_cerebellar_bundles():
    cp_rois = afd.read_cp_templates()
    cp_bundles = abd.BundleDict(
    {
        "ICP_L": {
            "include": [
                cp_rois["ICP_L_inferior_prob"],
                cp_rois["ICP_L_superior_prob"],
            ],
            "cross_midline": False,
        },
        "ICP_R": {
            "include": [
                cp_rois["ICP_R_inferior_prob"],
                cp_rois["ICP_R_superior_prob"],
            ],
            "cross_midline": False,
        },
        "MCP_L": {
            "include": [
                cp_rois["MCP_L_inferior_prob"],
                cp_rois["MCP_R_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_L_inter_prob"],
            ],
            "cross_midline": True,
        },
        "MCP_R": {
            "include": [
                cp_rois["MCP_R_inferior_prob"],
                cp_rois["MCP_L_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_R_inter_prob"],
            ],
            "cross_midline": True,
        },
        "SCP_L": {
            "include": [
                cp_rois["SCP_L_inferior_prob"],
                cp_rois["SCP_L_inter_prob"],
                cp_rois["SCP_R_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_L_superior_prob"],
            ],
            "cross_midline": True,
        },
        "SCP_R": {
            "include": [
                cp_rois["SCP_R_inferior_prob"],
                cp_rois["SCP_R_inter_prob"],
                cp_rois["SCP_L_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_R_superior_prob"],
            ],
            "cross_midline": True,
        },
    }
    return cp_bundles
)
```